osstracker-docs/README.md

```diff
@@ -7,3 +7,77 @@ High Level View
 ===============
 
 ![High Level View](https://raw.githubusercontent.com/Netflix/osstracker/documentation/osstracker-docs/HighLevelView.png)
+
+Manual installation
+===================
+
+Following manual installation is sample installation based on Linux system using docker containers. You can adapt it easily for other system using *docker-machine*, but care about `CASS_HOST` and `ES_HOST` they should be equals to `docker-machine ip MACHINE_NAME`.
+
+**ATTENTION** neither *Cassandra* nor *ElasticSearch* containers define mounting volumes, if containers are removed data will be lost.
+
+Clone OSS Tracker source code
+
+```
+git clone https://github.com/Netflix/osstracker.git
+```
+
+Run Cassandra instance (using docker)
+
+```
+export CASS_HOST=localhost
+export CASS_PORT=9042
+docker run -d --name osstracker-cassandra -p ${CASS_PORT}:9042 cassandra
+```
```

Comment:
I wonder if there is any cassandra tuning that can be done.  Running this locally, cassandra is very much slower than in our environment.  Need to look into cassandra logs to see if there are any memory issues.